src/components/Docs/SnippetViewer/CheckRequestViewer.tsx

```diff
@@ -40,14 +32,22 @@ ${
         allowed ? 'A green path from the user to the object' : 'A red object'
       } indicating that the response from the API is \`{"allowed":${allowed ? 'true' : 'false'}}\`
 `;
+    case SupportedLanguage.CLI:
+      return `fga query check --store-id=$FGA_STORE_ID --model-id=${modelId} ${user} ${relation} ${object}${
```

Comment:
do we need to refer to FGA_SERVER_URL?

Reply:
Don't think so - default server would work: `http://localhost:8080`

Setting up an alternate server is described in the setup